src/getting_started/installation_options.md

```diff
@@ -0,0 +1,272 @@
+
+To download a pre-built release of Parsec, you need to be running a 64-bit Linux system on the x86
+architecture, and you need to have at least version 2.27 of the Gnu C Library (`GLIBC`), which you
+can check by running the following command:
+
+```
+ldd --version
+```
+
+### Download the Latest Quick-Start Release Bundle
+
+Run the following command to download and unpack the `quickstart-1.1.0-linux_x86` folder. It
+contains the Parsec service (`parsec`), its pre-built configuration (`config.toml`) and the Parsec
+tool (`parsec-tool`).
+
+```
+curl -s -N -L https://github.com/parallaxsecond/parsec/releases/download/1.1.0/quickstart-1.1.0-linux_x86.tar.gz | tar xz
+```
+
+### Start the Parsec Service
+
+From within the `quickstart-1.1.0-linux_x86` directory, start the Parsec service with this command:
+
+```
+./parsec &
+```
+
+You should see some lines of console output as the service starts, ending with the following:
+
+```
+[INFO  parsec] Parsec is ready.
+```
+
+### Using the Parsec Tool
+
+As the Parsec socket file is at a non-default location, you will need to set the
+`PARSEC_SERVICE_ENDPOINT` environment variable first.
+
+```
+export PARSEC_SERVICE_ENDPOINT=unix:$(pwd)/parsec.sock
+```
+
+You can now use the `parsec-tool` to check that the service is running:
+
+```
+./parsec-tool ping
+```
+
+If the Parsec components are correctly downloaded and running, you should see output similar to the
+following:
+
+```
+[INFO] Service wire protocol version
+1.0
+```
+
+### Controlling the Service Manually
+
+When using the Parsec service as a pre-built download, it will not be installed as a system service.
+Therefore, to stop the service, issue the following command:
+
+```
+pkill parsec
+```
+
+You should see some lines of output ending with:
+
+```
+[INFO parsec] Parsec is now terminated.
+```
+
+You can also cause the service to restart, which can be useful if you have made some configuration
+changes for example. This command will cause the service to reload its configuration and restart:
+
+```
+pkill -SIGHUP parsec
+```
+
+Again, this will produce some lines of output, and the final line will be:
+
+```
+[INFO parsec] Parsec configuration reloaded.
+```
+
+### Running the Test Script
+
+The quick-start bundle also contains the `parsec-cli-tests.sh` testing script, which executes a
+simple set of tests to ensure that the Parsec service is operating correctly. Some of these tests
+use the local `openssl` installation as a point of comparison, ensuring that Parsec's results are
+equivalent to those expected by `openssl`.
+
+As this script uses the `parsec-tool`, the `PARSEC_SERVICE_ENDPOINT` environment variable needs to
+be set as follows:
+
+```
+export PARSEC_SERVICE_ENDPOINT="unix:$(pwd)/parsec.sock"
+```
+
+If parsec-tool is not installed into a directory included in `PATH`, then you also need to define
+`PARSEC_TOOL` environment variable with a full path to it:
+
+```
+export PARSEC_TOOL="$(pwd)/parsec-tool"
+```
```

Comment:
parsec-cli-tests.sh uses default value for and looking for  parsec-tool and openssl in PATH. These two (and OPENSSL) exports are required only in non-default cases.
I haven't checked fedora and opensuse packages buyt I would expect that they install and configure parsec and parsec-tool in default way

Reply:
Correct, but the quickstart package is intended as a self-standing blob (hence it packages both service and tool), so you can assume that the `parsec_tool` is _here_, and that the socket will also be in the current directory. We don't introduce those exports in the "generic" CLI test script, but we could dump them in before packaging the quickstart package is my point.

Reply:
Ah, I never liked that the quick start package uses non-default values.
But, AFAIU, this installation_options.md covers all installation cases not the quick start package only. So, it correctly explains what exports should be used in non-default cases.

Reply:
> But, AFAIU, this installation_options.md covers all installation cases not the quick start package only.

The whole file yes, but this specific section is under `Option 2: Download a Quick-Start Release`. Anyway, it was mostly a suggestion, don't think two exports add that much drag to the experience.

> Ah, I never liked that the quick start package uses non-default values.

Agree, but otherwise it would become quite an involved process of setting things up I think.

Reply:
@ionut-arm, there is no particular reason except maybe taking the script as is. However, since the package is for experimentation only and wants to deliver the smoothest experience. Then we might change the defaults for the `packaged` `parsec-cli-tests.sh` 
But again, as this export 
```export PARSEC_SERVICE_ENDPOINT="unix:$(pwd)/parsec.sock"``` 
is not avoidable if the user uses `parsec-tool` alone.